kriya-tech/src/ContactPage.jsx

```diff
@@ -0,0 +1,56 @@
+// src/ContactPage.jsx
+import React from 'react';
+
+function ContactPage() {
+  return (
+    <div className="min-h-screen flex items-center justify-center bg-gray-100">
+      <div className="bg-white p-8 rounded shadow-md w-96">
+        <h1 className="text-2xl font-semibold mb-4">Contact Us</h1>
+        <form>
+          <div className="mb-4">
+            <label htmlFor="name" className="block text-gray-600">
+              Name
+            </label>
+            <input
+              type="text"
+              id="name"
+              name="name"
+              className="w-full border border-gray-300 rounded px-3 py-2"
+            />
+          </div>
+          <div className="mb-4">
+            <label htmlFor="email" className="block text-gray-600">
+              Email
+            </label>
+            <input
+              type="email"
+              id="email"
+              name="email"
+              className="w-full border border-gray-300 rounded px-3 py-2"
+            />
+          </div>
+          <div className="mb-4">
+            <label htmlFor="message" className="block text-gray-600">
+              Message
+            </label>
+            <textarea
+              id="message"
+              name="message"
+              className="w-full border border-gray-300 rounded px-3 py-2"
+            ></textarea>
+          </div>
+          <div className="text-center">
+            <button
+              type="submit"
+              className="bg-blue-500 text-white px-4 py-2 rounded hover:bg-blue-600"
```

Comment:
Color names like text-white, border-gray-300, text-gray-600 is hardcoded. 
What if some day we want to change the color schema or color palette of the website or of this particular page?
Can you not name these colors somewhere in a color config file and use the variable names of these colors to be used in this page by importing them?

Reply:
This is applicable throughout the entire project and this PR as well. You can name these like variables like formBackgroundColor, formWidthBorderColor, formHeightBorderColor, etc.